docs/gitbook/tutorials/gloo-progressive-delivery.md

```diff
@@ -1,12 +1,15 @@
 # Gloo Canary Deployments
 
-This guide shows you how to use the [Gloo](https://gloo.solo.io/) ingress controller and Flagger to automate canary deployments.
+This guide shows you how to use the [Gloo Edge](https://gloo.solo.io/) ingress controller and Flagger to automate canary deployments.
 
 ![Flagger Gloo Ingress Controller](https://raw.githubusercontent.com/weaveworks/flagger/master/docs/diagrams/flagger-gloo-overview.png)
 
 ## Prerequisites
 
-Flagger requires a Kubernetes cluster **v1.11** or newer and Gloo ingress **1.3.5** or newer.
+This guide was written for Flagger version **1.5.0** or higher. Prior versions of Flagger used Gloo upstream groups to handle
+canaries, but newer versions of Flagger use Gloo route tables to handle canaries as well as A/B testing.
+
+Flagger requires a Kubernetes cluster **v1.11** or newer and Gloo Edge ingress **1.6.0** or newer.
 
 Install Gloo with Helm v3:
```

Comment:
note, the guide will not work as written until 1.6.0 is released in two weeks, as `helm install` installs the latest 1.5 release by default.

I can provide the `--version` flag to the `helm install` command if we want to hard-code the guide to a 1.6 version (for now, a beta). Thoughts?